pages/a-la-francaise.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="utf-8" />
+    <title>À la Française</title>
+    <meta name="description" content="Page menu du restaurant À la française" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"
+      integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Shrikhand&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <header>
+    <a href="../index.html"
+      ><i class="fas fa-arrow-left"></i>
+      <h1>Ohmyfood</h1></a
+    >
+  </header>
+  <main>
+    <img
+      src="../assets/images/restaurants/francaise.jpg"
+      alt="table de restaurant garnie d'assiettes gastronimiques"
+    />
+    <section id="menu">
+      <h2>À la française<i class="far fa-heart"></i></h2>
+      <article>
```

Comment:
add a new section with id entries and put three article inside. Idem for plat and dessert